### ISSUE TITLE
Wasn't there originally a note column?

### ISSUE THREAD

**Issue by ladlon** [turtletooth/tahoma2d#433]
I love the Note feature, although it would be great to be able to put notes/markers on the actual time bar (as often the notes get in the way when you are trying to view the keys).

Ideally, you'd have a dedicated marker/note bar going along the top of the timeline which would remain there (under the time bar) and not scroll off the display when the columns are scrolled (so they are always visible).

Maybe, too, optionally allow the adjustment of their width, so that (optionally) their text is more meaninful, as the text gets really truncated with the current version.

**Comment by manongjohn** [comment]
`Level -> New -> New Note Level`

Or 

Right-click column headers and select `New Note Level`

**Comment by ladlon** [comment]
Huh!  Weird...  The New Level button on the top of the timeline (which is what I normally use) doesn't have that option.

Okay, well at least it's there.  Maybe it should be added to the window launched by the New Level button on the timeline?

**Comment by artisteacher** [comment]
The new level dialog has never had the ability to create a note level, as far as I know.

**Comment by manongjohn** [comment]
Yeah, it never was there.  Note Levels were originally only for Lip syncing.  When it was opened up to be a free text cell for other users, I don't recall why it wasn't added to the New Level dialog.

I think it should be there as well.

**Comment by ladlon** [comment]
It would be good if the Note column could be optionally 'locked' to the top of the timeline, so it doesn't scroll off the display.

Granted, you can manually keep moving it to be beside the track you want to work on while referencing it.... but (besides the manual work), it also would currently break any expressions referencing other columns, due to the current issue with column names changing as they get shifted around (and the names not getting updated in the expression).

**Comment by ladlon** [comment]
Odd that the current version of the Note column doesn't display the full note text when you expand the range of the note cell.  It keeps it truncated (to one letter?!), even if the 'cell' is expanded to span over several cells.  I understand the reasoning for the XSHeet (since it's vertical), but for the timeline, the expanded span is a perfect opportunity to display the full text.

**Comment by artisteacher** [comment]
If you are using the note column with the Text FX, the position matters.

**Comment by ladlon** [comment]
I'm not familiar (haven't used) the Text FX...  I just added a Note column, clicked in a cell, and typed a few words.
The text (understandably) gets truncated when it's a single cell span... but when I extend the span across several cells, it still stays truncated, despite there being more room for more text.

**Comment by artisteacher** [comment]
If you pair a note level with the Text Iwa FX, the text displays in the render - great for captions, subtitles & the like.

**Comment by ladlon** [comment]
Oh, okay, cool...  Ya, I'm talking about just for timeline use... although that's a good way to put notes in the actual render, ya!
